sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -120,12 +122,71 @@ export function useFormatters() {
     };
   };
 
+  /**
+   * Converts a string flag defined as Y/N to Yes/No.
+   * @param yesNoFlag Y/N flag.
+   * @returns Yes, No or a default empty value case not a expected flag value as Y or N.
+   */
+  const yesNoFlagDescription = (yesNoFlag?: string): string => {
+    if (!yesNoFlag) {
```

Comment:
The logic can be simplified using mapper.
`const flagMapper = {y:"Yes",n:"No"};

return flagMapper[yesNoFlag?.toLowerCase()] || DEFAULT_EMPTY_VALUE;`

Reply:
It actually needs to be as below. Maybe it is less code but not easier to read, IMO.
![image](https://user-images.githubusercontent.com/61259237/175365991-5abc0561-e591-4e62-9cc5-10afbb10f2be.png)
I would prefer to keep the first check in place as below:
![image](https://user-images.githubusercontent.com/61259237/175366364-d47ca802-3661-4385-89bd-4f03d1f465de.png)